docs/assets/documents/governance-for-hospitalNotification.md

```diff
@@ -0,0 +1,111 @@
+# Governance for MedCom HospitalNotification
+
+* [1 Exchange of HospitalNotification message](#1-exchange-of-hospitalnotification-message)
+  * [1.2 VANS-network and Fælleskommunal infrastructure](#12-vans-network-and-f-lleskommunal-infrastructure)
+    * [Type A](#type-a)
+    * [Type B](#type-b)
+    * [Type C](#type-c)
+  * [1.3 Acknowledgements](#13-acknowledgements)
+  * [1.4 Envelopes](#14-envelopes)
+* [2 HospitalNotification message](#2-hospitalnotification-message)
+  * [2.1 MedComHospitalNotificationMessage profile](#21-medcomhospitalnotificationmessage-profile)
+  * [2.2 MedComHospitalNotificationEncounter profile](#22-medcomhospitalnotificationencounter-profile)
+  * [2.3 MedComMessagingProvenance](#23-medcommessagingprovenance)
+<br><br>
+
+## 1 Exchange of HospitalNotification message 
+
+The flow of events of a HospitalNotification message differs from the regular message exchange over the VANS-network, as a message is exchange over the VANS-network and KOMBITs-network. Questions regarding KOMBITs infrastructure, called Fælleskommunal infrastructure, and KOMBIT's envelope, called Beskedfordeler-envelope, should be addressed to KOMBIT. 
+
+
+### 1.2 VANS-network and Fælleskommunal infrastructure
+The following sections describe the flow of a HospitalNotification message. There are three types of receiving systems that can receive a HospitalNotification: Type A, B and C. Since there is different requirements to acknowledging of a HospitalNotification, they are presented seperately.  
```

Comment:
Since there is different requirements to acknowledging of a HospitalNotification, they are presented seperately.  
erstattes af
Since there are different requirements to acknowledging a HospitalNotification, they are presented seperately.